tools/get_dicom.pl

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/perl -w 
+use strict;
+
+=pod
+
+=head1 NAME
+
+get_dicom.pl - extracts DICOM files for specific patient names/scan types
+
+=head1 SYNOPSIS
+
+perl get_dicom.pl [-p profile] [-n patient_name_patterns] [-t scan_type_patterns] [-d tmp_dir] [-o tarBasename]
+
+Available options are:
+
+-p      : name of the config file in C<../dicom-archive/.loris_mri> (defaults to
+          prod).
+-n      : comma separated list of MySQL patterns for the patient names that a DICOM file
+          has to have in order to be extracted. A DICOM file only has to match one of the 
+          patterns to be extracted. If no pattern is specified, then the patient name is 
+          not used to determine which DICOM files to extract.
+-t      : comma separated list of MySQL patterns of the acquisition protocols (scan types
+          names) that a DICOM file has to have in order to be extracted. A DICOM file only
+          has to match one of the patterns to be extracted. If no pattern is specified, then
+          the scan type name is not used to determine which DICOM files to extract.
+-d      : extract the files in directory <dir_argument>/get_dicom.pl.<UNIX_process_number>
```

Comment:
rename the directory to get_dicom_file.pl if suggestion above accepted.

Reply:
also, `<dir_argument>/get_dicom.pl.<UNIX_process_number>` should be in code format `C<<dir_argument>/get_dicom.pl.<UNIX_process_number>>`

Reply:
Done.